src/utils/promise-finally.js

```diff
@@ -0,0 +1,13 @@
+export default function (promise, onFinally) {
```

Comment:
Refactor it's to the external micromodule and don't return the promise, it will make code much cleaner:
```js
var promise = createPromise();

onceDone(promise, doSomething);
```